apps/artist/queries/overview.coffee

```diff
@@ -15,6 +15,14 @@ module.exports =
       artists (size: 15) @include(if: $artists){
         ... artistCell
       }
+      cta_artist: artists(size: 1) {
```

Comment:
We only load related artists client-side, so added the one we want here rather than the bootstrapped server-side call.